app/javascript/components/search/controls/download/DownloadSelectionModal.js

```diff
@@ -1,36 +1,83 @@
 import React, { useEffect, useState } from 'react'
 import Modal from 'react-bootstrap/lib/Modal'
+import _cloneDeep from 'lodash/cloneDeep'
 
 import DownloadCommand from './DownloadCommand'
 import DownloadSelectionTable, {
-  newSelectedBoxesState, getSelectedFileIds, getSelectedFileStats, bytesToSize
+  newSelectedBoxesState, getSelectedFileHandles, getSelectedFileStats, bytesToSize
 } from './DownloadSelectionTable'
-import { fetchDownloadInfo } from 'lib/scp-api'
 
+import { fetchDownloadInfo, fetchDrsInfo } from 'lib/scp-api'
+
+const TDR_COLUMNS = ['analysis', 'sequence']
+const SCP_COLUMNS = ['matrix', 'metadata', 'cluster']
 
 /**
   * a modal that, given a list of study accessions, allows a user to select/deselect
   * studies and file types for download.  This queries the bulk_download/summary API method
   * to retrieve the list of study details and available files
   */
-export default function DownloadSelectionModal({ studyAccessions, show, setShow }) {
+export default function DownloadSelectionModal({ studyAccessions, tdrFileInfo, show, setShow }) {
   const [isLoading, setIsLoading] = useState(true)
+  const [isLoadingTDR, setIsLoadingTDR] = useState(true)
   const [downloadInfo, setDownloadInfo] = useState([])
   const [selectedBoxes, setSelectedBoxes] = useState()
+  const [downloadInfoTDR, setDownloadInfoTDR] = useState([])
+  const [selectedBoxesTDR, setSelectedBoxesTDR] = useState()
   const [stepNum, setStepNum] = useState(1)
 
+  const scpAccessions = studyAccessions.filter(accession => accession.startsWith('SCP'))
+  const tdrAccessions = studyAccessions.filter(accession => !accession.startsWith('SCP'))
```

Comment:
Is it safe to assume if not `SCP` will be `TDR`? If the `TDR` accessions start with `TDR` could we look for that instead so if we add more types in the future it's a bit safer?

Reply:
`TDR` accessions don't have a naming scheme, only `SCP` ones.  We're shoe-horning in the "project short name" as an accession here just to make the code easier to work with.

Reply:
oh okay, thanks for the context 👍 this comment can be considered resolved then